benchmarks/utils/join.py

```diff
@@ -0,0 +1,62 @@
+import pandas as pd
+
+
+def fetch_data(dataset_name):
+    """Fetch datasets from https://github.com/Yeye-He/Auto-Join/tree/master/autojoin-Benchmark
+    """
+    repository = "Yeye-He/Auto-Join"
+    if isinstance(dataset_name, str):
+        dataset_name = dataset_name.replace(' ', '%20')
+        base_url = f"https://raw.githubusercontent.com/{repository}/master/autojoin-Benchmark/{dataset_name}"  # noqa
```

Comment:
Can be broken into subparts to avoid using noqa (nitpick)
```suggestion
        base_url = (
            f"https://raw.githubusercontent.com/"
            f"{repository}/master/autojoin-Benchmark/{dataset_name}"
        )
```